_pages/algorithm-development/advanced-algorithm-development/dealing-with-bottlenecks.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: article
+title:  "Algorithm Bottlenecks: How to get the most out of your algorithms"
+permalink: algorithm-development/advanced-algorithm-development/list-packages/
+excerpt: "Examples on how to remove performance bottlenecks, from bandwidth to compute"
+categories: [algo-dev]
+tags: [algo-dev]
+show_related: true
+author: james_sutton
+image:
+    teaser:  /post_images/batch-processing/batch_processing.png
+---
+
+# Algorithm Parallelism and Threading Tutorial
+
+This tutorial is primarily targetted at an advanced audience looking for solutions to improve runtime performance of already existing algorithms by removing bottlenecks.
+ If you'd like to find the introductory tutorials please check [this page](https://algorithmia.com/developers/algorithm-development/advanced-algorithm-development) first.
```

Comment:
Please change to {{site.baseurl}}/algorithm-development/advanced-algorithm-development